wazo_confd/plugins/outcall/validator.py

```diff
@@ -15,5 +14,4 @@ def build_validator():
         create=[
             UniqueField('name', lambda name: outcall_dao.find_by(name=name), 'Outcall')
```

Comment:
I would remove this validator too, since the input does not come from the user anymore